undyingkingdoms/blueprints/game_clock/models.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+
+import jwt
+from flask import current_app
+
+from extensions import flask_db as db
+from undyingkingdoms.models.bases import GameState
+
+
+class Token(GameState):
+    @staticmethod
+    def encode_auth_token():
+        """Generate the authentication token.
+
+        :return: string
+        """
+
+        try:
+            payload = {
+                'exp': datetime.datetime.utcnow() + datetime.timedelta(days=365),
+                'iat': datetime.datetime.utcnow(),
+                'sub': current_app.config.get('CLOCK_KEY')
+            }
+            return jwt.encode(
+                payload,
+                current_app.config.get('SECRET_KEY'),
+                algorithm=current_app.config.get('JWT_ALGORITHM')
+            )
```

Comment:
This code should probably generate a random token and store it in a list of valid tokens.